panther_gazebo/config/led_strips.yaml

```diff
@@ -0,0 +1,11 @@
+chanel_1:
+  position: [0.362, 0.0, 0.201]
+  led_strip_width: 0.5
```

Comment:
can you explain why this is 0.5, because it's a bit abstract to me.

Reply:
It's true length on panther led strip

Reply:
Ahh, right